lsst-epo/investigations-client
# [F Save work flow

JIRA: https://jira.lsstcorp.org/browse/EPO-8656

## What this change does ##

A few sentences describing the outcome of the change (“fixes a thing”; "adds the stuff") and how you tackled it (“added a prop to prevent lazyloading in the presence of barbara walters”).  How successful is the change?  And how are you measuring that success?

## Notes for reviewers ##

Client branch only, no updates to API or database. 

Include an indication of how detailed a review you want on a 1-10 scale.
- 3

## Testing ##

- Start either logged in or out and complete up to Colorizing the Images in Coloring the Universe
- Click the save button
- A toast should appear that indicates save status. If successful but not logged in, additional buttons should appear that direct users to log in
- Error statuses can be tested by editing the save actions to always return errors.

## Checklist ##

If any of the following are true, please check them off

## Screenshots (optional) ##

![image](https://github.com/lsst-epo/investigations-client/assets/11721838/c6c86c94-c070-449b-8c02-c34f34397fad)
